.eslint-dictionary.json

```diff
@@ -49,7 +49,9 @@
   "iops",
   "iteratively",
   "javascript",
+  "jose",
   "jsdoc",
+  "jwks",
```

Comment:
nit: for this is probably better to add a lint ignore that adding to the dictionary

Reply:
Ideally our spell checker would allow names of dependencies or exports of dependencies